Casino.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+// Load in remix: remix.loadurl("https://github.com/web3examples/ethereum/solidity_examples/Casino.sol")
+// Carefull with gas-estimates: due to random this doesn''t allways work
+pragma solidity >=0.7.0 <0.9.0;
```

Comment:
This allows for an old version of solidity!

Reply:
Documented in https://github.com/gpersoon/issue-system/issues/3

So the conversion can be resolved